test/workflow-status/testpipeline.js

```diff
@@ -0,0 +1,41 @@
+"use strict"
+
+const {
+  task,
+  join
+} = require("../..")
+const {
+  assert
+} = require("chai");
+const mocha = require("mocha")
+
+const {
+  store
+} = require("../..")
+var expect = require("chai").expect
+
+const anotherTask = task({
+    name: "Run a JS file",
+    output: "ran.txt"
+  },
+  () =>
+  `node /home/evoxtorm/Desktop/Bionode-watermill/bionode-watermill/test/workflow-status/wait-error.js && touch ran.txt`
```

Comment:
`node ./test/workflow-status/wait-error.js` should also work, if `mocha` is being called from at the root of the repo

Reply:
does `node ${path.resolve(__dirname, 'wait-error.js')} && touch ran.txt` work?